src/main/resources/db/migration/local/R__4_create_bookings_for_cas3_south_west.sql

```diff
@@ -50,6 +50,486 @@ VALUES
   )
 ON CONFLICT(id) DO NOTHING;
 
+--- Add a Booking arriving soon ---
+
+INSERT INTO
+  bookings (
+    "id",
+    "arrival_date",
+    "departure_date",
+    "crn",
+    "original_arrival_date",
+    "original_departure_date",
+    "premises_id",
+    "bed_id",
+    "service",
+    "created_at",
+    "noms_number"
+  )
+VALUES
+  (
+    'c628b111-d1c4-49b5-9b7a-2f4aad9c2755',
+    CURRENT_DATE + (4 * 2) + 1,
```

Comment:
We mostly copy these these statements but we push the arrival and departure dates forward on each one so they don't overlap.